app/src/ui/tutorial-panel/tutorial-panel.tsx

```diff
@@ -0,0 +1,165 @@
+import * as React from 'react'
+import { join } from 'path'
+import { LinkButton } from '../lib/link-button'
+import { Button } from '../lib/button'
+import { Monospaced } from '../lib/monospaced'
+import { Repository } from '../../models/repository'
+import { Dispatcher } from '../dispatcher'
+
+interface ITutorialPanelProps {
+  readonly dispatcher: Dispatcher
+  readonly repository: Repository
+  readonly externalEditorLabel?: string
+}
+
+export class TutorialPanel extends React.Component<
+  ITutorialPanelProps,
+  { openId: string }
+> {
+  public constructor(props: ITutorialPanelProps) {
+    super(props)
+    this.state = { openId: 'step-1' }
+  }
+
+  private openFileInEditor = () => {
+    this.props.dispatcher.openInExternalEditor(
+      join(this.props.repository.path, 'README.md')
+    )
+  }
+
+  private openPullRequest = () => {
+    this.props.dispatcher.createPullRequest(this.props.repository)
+  }
+
+  public render() {
+    return (
+      <div id="tutorial" className="panel">
+        <ol>
+          <ListItem
+            summaryText="Install a text editor"
+            id="step-1"
+            openId={this.state.openId}
+            onClick={this.handleToggle}
+          >
+            <div>
+              It doesn’t look like you have a text editor installed. We can
+              recommend{' '}
+              <LinkButton uri="https://atom.io" title="Open the Atom website">
+                Atom
+              </LinkButton>
+              {` or `}
+              <LinkButton
+                uri="https://code.visualstudio.com"
+                title="Open the VS Code website"
+              >
+                Visual Studio Code
+              </LinkButton>
+              , but feel free to use any.
+            </div>
+            <LinkButton>I have an editor</LinkButton>
+          </ListItem>
+          <ListItem
+            summaryText="Make a branch"
+            id="step-2"
+            openId={this.state.openId}
+            onClick={this.handleToggle}
+          >
+            <div>
+              Create a branch by going into the branch menu in the top bar and
+              clicking New Branch.
+            </div>
+            <kbd>⇧⌘N</kbd>
+          </ListItem>
+          <ListItem
+            summaryText="Edit a file"
+            id="step-3"
+            openId={this.state.openId}
+            onClick={this.handleToggle}
+          >
+            <div>
+              Open this repository in your preferred text editor. Edit the{' '}
+              <Monospaced>README.md</Monospaced> file, save it, and come back.
+            </div>
+            <Button
+              onClick={this.openFileInEditor}
+              disabled={!this.props.externalEditorLabel}
+            >
+              Open Editor
```

Comment:
This should be title case on macOS and sentence case on Windows.
```suggestion
              {__DARWIN__ ? 'Open Editor' : 'Open editor' }
```